File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int ln= nums.length;
        int count=0;
        for(int i=0; i<ln; i++){
            if(nums[i]>nums[(i+1)%ln]){
                count=count+1;
            }
        if(count>1){
            return false;
        }
        }
        return true;
    }
        
}

File: 1752-check-if-array-is-sorted-and-rotated/1752-check-if-array-is-sorted-and-rotated.py
class Solution(object):
    def check(self, nums):
        sz=len(nums)
        a=0
        for i in range(sz-1):
            if nums[i]>nums[i+1]:
                a+=1
        if nums[sz-1]>nums[0]:
            a+=1
        return a<=1
        